utils/cliutils/commandsflags.go

```diff
@@ -1779,6 +1821,26 @@ var commandFlags = map[string][]string{
 		setupFormat,
 	},
 	Intro: {},
+	// Pipelines commands
+	Status: {
+		branch, serverId, name, monitor, multiBranch,
+	},
+	Trigger: {
+		branch, serverId, pipelineName, multiBranch,
+	},
+	Pip: {},
+	Validate: {
+		Resources, serverId,
+	},
+	Version: {
+		serverId,
+	},
+	Sync: {
+		branch, repository, serverId,
+	},
+	SyncStatus: {
+		branch, repository, serverId,
+	},
```

Comment:
Would you like to support only serverId at this stage? The other commands support user, username, password, etc...

Reply:
For now yes with minimal support. I will add support for other flags very soon